### PR TITLE
Update to polyfill-service@2.1.0 & default to using polyfill

### PR DESCRIPTION
**Pull request by jbuck**

### CODE REVIEW

File: server.js [hunk 1]
```diff
@@ -180,11 +180,12 @@ server.route([
       var polyfills = PolyfillSet.fromQueryParam(features || 'default', flags);
       var params = {
         features: polyfills.get(),
-        minify: true
+        minify: true,
+        unknown: 'polyfill'
```
Comment (jbuck) [inline server.js]: This changes the behaviour of the polyfill service to default to using all of the polyfills when it cannot detect the user-agent correctly.

**Comment by jbuck** [comment]
@ScottDowne r?

**Comment by jbuck** [comment]
Fixes #691

**Comment by ScottDowne** [comment]
R+

**Pull request merged**